test.log

```diff
@@ -1,3257 +0,0 @@
-=== RUN   TestEncodeBlockAnnounce
-=== PAUSE TestEncodeBlockAnnounce
```

Comment:
:smile: WHY WAS THIS COMMITTED :laughing:

Reply:
removed lol